corebuild/integration/ILLink.Tasks/ILLink.Tasks.targets

```diff
@@ -287,38 +317,27 @@
                               RootDescriptorFilePath="$(_IntermediateRootDescriptorPath)" />
   </Target>
 
+  <!-- Uses _PublishConflictPackageFiles, an input to the publish deps
+       file generation target, to exclude the removed managed
+       assemblies and native dependencies from the generated deps
+       file. -->
+  <UsingTask TaskName="ComputeRemovedAssemblies" AssemblyFile="$(LinkTaskDllPath)" />
+  <Target Name="_ExcludeRemovedFilesFromDepFileGeneration"
+          DependsOnTargets="_ComputeManagedResolvedAssembliesToPublish;_ComputeLinkedAssemblies;_FindNativeDeps;_HandlePublishFileConflicts"
```

Comment:
Some of the dependencies here are also redundant.

Reply:
Here I'd prefer to be explicit. For each itemgroup that this task uses, I added a dependency on the target that produces it, so that the inputs will be available even if some of the dependencies between the dependent targets change.